platform/android/export/export_plugin.cpp

```diff
@@ -372,7 +373,7 @@ void EditorExportPlatformAndroid::_check_for_changes_poll_thread(void *ud) {
 
 			ea->device_lock.unlock();
 		}
-
+#endif
```

Comment:
Nitpick but keeping a line between the two blocks of code would be nice.

Reply:
Done!